Negotiation/FormatNegotiator.php

```diff
@@ -40,7 +40,7 @@ public function add(RequestMatcherInterface $requestMatcher, array $options = []
         $this->map[] = [$requestMatcher, $options];
     }
 
-    public function getBest($header, array $priorities = [])
+    public function getBest($header, array $priorities = [], $strict = false)
```

Comment:
seems to be an unnecessary change

Reply:
it is a necessary change as `willdurand/negotiation` changed the signature of this method in v3 and we have to update it too to comply with the change.

Reply:
Oh, ups. I checked the complete codebase ... but missed to check the base class. Thanks for the explanation & sorry for stealing your time

Reply:
No problem, it's very nice of you to review these changes :)

Reply:
The least I can do if relying on this bundle^^ and btw: my projects api test suite is green with this branch (around 100 integration tests). 
Looking forward for a PHP 8 enabled release! 😁

Reply:
ah yeah sorry for the breaking change here.. hopefully it wasn't too complicated to migrate

Reply:
No worry, it was manageable :) Thank you for your work on this library!